lib/symmetric_encryption/cipher.rb

```diff
@@ -130,7 +130,8 @@ def self.generate_random_keys(params = {})
         encrypted_key               = key_encryption_key.encrypt(key)
         cipher_conf[:encrypted_key] = SymmetricEncryption::Encoder[encoding].encode(encrypted_key)
       else
-        cipher_conf[:key] = key.to_s
+        params.delete(:key)
+        cipher_conf[:key] = SymmetricEncryption::Encoder[encoding].encode(key.to_s)
```

Comment:
After your comment, I notice the class `Encoder`. So I just replaced the `Base64` for your class.